hbase-endpoint/pom.xml

```diff
@@ -47,24 +47,6 @@
           <skipAssembly>true</skipAssembly>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.xolstice.maven.plugins</groupId>
-        <artifactId>protobuf-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>compile-protoc</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>compile</goal>
-            </goals>
-             <configuration>
-              <additionalProtoPathElements>
-                <additionalProtoPathElement>${basedir}/../hbase-protocol/src/main/protobuf</additionalProtoPathElement>
-              </additionalProtoPathElements>
-             </configuration>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
It doesn't have pb local anymore?

Reply:
All in hbase-protocol-shaded, as we need to some relocation and replacement in code.